SpatialGDK/Source/Private/Interop/SpatialOutputDevice.cpp

```diff
@@ -20,6 +24,11 @@ FSpatialOutputDevice::~FSpatialOutputDevice()
 
 void FSpatialOutputDevice::Serialize(const TCHAR* InData, ELogVerbosity::Type Verbosity, const class FName& Category)
 {
+	if (!WorkerLogFile.IsEmpty())
```

Comment:
Shouldn't we do this after the verbosity check to make sure we don't log more than the user wants?